### PR TITLE
Improvements to AI page

### PR DESCRIPTION
**Pull request by andriyParashchuk**
Staging - https://jetthoughts-pr-370.surge.sh/agentinbox

### CODE REVIEW

**Comment by obliviusm** [comment]
Deployed to https://jetthoughts-pr-370.surge.sh

**Comment by andriyParashchuk** [comment]
`Remove line (iPhone 6/7/8 Plus)` - This is not a bug. If you testing on chrome device toolbar need set zoom to `100%`.

**Comment by andriyParashchuk** [comment]
@obliviusm @pftg I fixed remarks above. Also added slide switching with mouse wheel.
Please review

**Comment by sweatC** [comment]
1. Add menu icon. 2. Text has 3 rows you have 4 rows.
**Design:**
image
**Site:**
image
3. Check the font-weight properly. It must be thinner. Check all sections.
**Design:**
image
**Site:**
image
image
image
image
image
image
4. Check number of rows. Must be 6.
**Design:**
image
**Site:**
image

**Comment by andriyParashchuk** [comment]
@sweatC It's good that you noticed the difference in the font. For font, there is a task #351. 
Please check text rows on resolution with width 1600px like the design. I hope you will not find a difference)
Also I would no attach importance, because text will changes.

File: _includes/agentinbox/ai-projects.html [hunk 1]
```diff
@@ -5,26 +5,26 @@ <h2 class="ai-projects__title">
...
       <li class="list__item fade-right delay-4">
-        <a href="#" class="ai-projects__link">
+        <a href="https://www.openapply.com" class="ai-projects__link" target="_blank" rel="noreferrer">
           <strong class="ai-projects__name">Openapply</strong>
           <small class="ai-projects__development">web development</small>
         </a>
       </li>
       <li class="list__item fade-right delay-5">
-        <a href="#" class="ai-projects__link">
+        <a href="https://mobilecoach.com" class="ai-projects__link" target="_blank" rel="noreferrer">
```
Comment (pftg) [inline _includes/agentinbox/ai-projects.html:27]: please use https://github.com/jetthoughts/jetthoughts.github.io/blob/develop/_data/use_cases.yml for storing urls

Comment (andriyParashchuk) [reply _includes/agentinbox/ai-projects.html:27]: Maybe let's create a separate task for this?

Comment (andriyParashchuk) [reply _includes/agentinbox/ai-projects.html:27]: this problem is on the also main page

**Review by sweatC — APPROVED** [review]
LGTM

**Comment by sweatC** [comment]
@pftg pls merge if all is OK

**Pull request merged**